easybuild/easyblocks/generic/scons.py

```diff
@@ -54,7 +54,8 @@ def test_step(self):
         Test with SCons 
         """
         cmd = "scons check"
-        (out, _) = run_cmd(cmd, log_all=True)
+        if self.cfg['runtest']:
+            (out, _) = run_cmd(cmd, log_all=True)
```

Comment:
would there be any harm in doing it exactly like in the `ConfigureMake` block?
```python
        if self.cfg['runtest']:
            cmd = "make %s" % (self.cfg['runtest'])
            (out, _) = run_cmd(cmd, log_all=True, simple=False)
```
It might come in handy later on?